### PR TITLE
AT1-217: Exception handling for CentralNode, SubarrayNode, DishLeafNode, CspMasterLeafNode and DishMaster

### PR DESCRIPTION
**Pull request by jyotinpo**
This pull request incorporates:
- Catch specific exceptions for CentralNode, SubarrayNode, DishLeafNode, and CspMasterLeafNode.
- Throw exceptions to clients.
- Resolve the issue in generating a test coverage report for CspMasterLeafNode.
- Resolve `None` FQDN error during test case execution of CspMasterLeafNode and DishLeafNode.

### CODE REVIEW

**Comment by jyotinpo** [comment]
Done. The commit id is: https://github.com/ska-telescope/tmc-prototype/commit/15c1ffe39e7b5598e869662300f289139bc7cefd

**Comment by jyotinpo** [comment]
Done. THe commit id is: https://github.com/ska-telescope/tmc-prototype/commit/15c1ffe39e7b5598e869662300f289139bc7cefd

File: tmcprototype/CentralNode/CentralNode/CentralNode.py [hunk 1]
```diff
@@ -363,12 +372,12 @@ def StandByTelescope(self):
         for name in range(0, len(self._dish_leaf_node_devices)):
             try:
                 self._leaf_device_proxy[name].command_inout(CONST.CMD_SET_STANDBY_MODE)
-            except Exception as except_occured:
+            except DevFailed as dev_failed:
                 print(CONST.ERR_EXE_STANDBY_CMD, self._dish_leaf_node_devices[name])
                 self._read_activity_message = CONST.ERR_EXE_STANDBY_CMD + str(self._dish_leaf_node_devices[
```
Comment (adityadangeska) [inline tmcprototype/CentralNode/CentralNode/CentralNode.py]: Check formatting.

Comment (jyotinpo) [reply tmcprototype/CentralNode/CentralNode/CentralNode.py]: Seems to be okay.

**Pull request merged**